src/lib/containers/UserSearchBoxV2.tsx

```diff
@@ -86,6 +87,7 @@ const UserSearchBoxV2: React.FC<UserSearchBoxProps> = props => {
     filterPredicate,
     typeFilter,
     placeholder,
+    focusOnSelect,
```

Comment:
Setting a default value here shouldn't affect the logic but it's good practice to not allow a null/undefined value if it has no meaning
```suggestion
    focusOnSelect = false,
```